test/fixtures/track-basic.json

```diff
@@ -32,7 +32,7 @@
       "$browser": "Safari",
       "$os": "iPhone OS",
       "$os_version": "8.1.3",
-      "$browser_version": "7.1.0.7",
+      "$browser_version": 7.1,
```

Comment:
The client-side library matches any version string that has 1 or more minor-versions (.x), and then parses the major and first-minor version as a number. I doubt the extra version numbers are useful, and even if they are, anything coming from a browser wouldn't have had them anyways.